app/models/user.rb

```diff
@@ -58,5 +59,6 @@ def avatar_attached?
 
   def presenter_class
     UserPresenter
+    InvitePresenter
```

Comment:
This is wrong. 
1. Why user model should use invite presenter?
2. Have you even run specs? This will broke other things.

Reply:
Use `UserPresenter`

Reply:
done